mod/bootstrap/set/type/bootswatch_skin.rb

```diff
@@ -1,15 +1,174 @@
-include_set Abstract::BootswatchTheme
+# Bootswatch themes are free themes for bootstrap available at https://bootswatch.com/.
+# For every bootswatch theme we have one card of card type "bootswatch skin".
+# They all have codenames following the pattern "#{theme_name}_skin".
+#
+# The original bootswatch theme is build from two files, `_variables.scss` and
+# `_bootswatch.scss`. The original bootstrap scss has to be put between those two.
+# `_variables.scss` overrides bootstrap variables, `_bootswatch.scss` overrides
+# bootstrap css (variables are defined with `!default` hence only the first appearance
+# has an effect, for css the last appearance counts)
+#
+# The content of a bootswatch theme card consists of four parts:
+#   * pre_variables: hard-coded theme independent stuff
+#       and bootstrap functions to make them available in the variables part
+#   * variables: the content from `_variables.scss`,
+#   * post_variables: the bootstrap css and libraries like select2 and
+#       bootstrap-colorpicker that depend on the theme
+#   * stylesheets: the content from `_bootswatch.scss` and custom styles
+#
+# For the original bootswatch themes all those parts are hard-coded and the content
+# is taken from files.
+# The bootswatch theme content is taken directly from the files in the bootswatch
+# submodule. For the rest we use code file cards.
+# Cards of type "customized bootswatch skin" have the same structure but make the
+# variables and stylesheets part editable.
+#
+# Bootswatch theme cards are "asset inputters," meaning they generate a
+# Machine cards usually store all involved input cards of all nested levels in
```

Comment:
old wording - machines are gone

Reply:
Right. I started in on this edit but never finished it. Thanks.